en/config/actuators.md

```diff
@@ -1,96 +1,250 @@
+  Includes V-Tail etc.
+
+
+### Motor Tilt Servo Geometry
+
+[VTOL tiltrotor vehicles](../frames_vtol/tiltrotor.md) can tilt their motors to transition between hover and forward flight.
+This section defines the properties of the tilting servos.
+These are mapped to specific motors in the [motor geometry for a tiltrotor](../config/actuators.md#motor-geometry-vtol-tiltrotor).
+
+The example below shows the tilt servo setup for the [tiltrotor motor geometry shown above](../config/actuators.md#motor-geometry-vtol-tiltrotor).
+
+![Tilt Servo Geometry Setup Example](../../assets/config/actuators/tilt_servo_geometry_config.png)
+
+The values that can be set are:
+- `Tilt servos`: The number of servos (tiltable motors).
+- `Angle at min tilt`: [Maximum tilt angle](#tilt-servo-coordinate-system) in degrees, relative to the z axis.
+- `Angle at max tilt`: [Minimum tilt angle](#tilt-servo-coordinate-system) in degrees, relative to the z-axis.
+- `Tilt direction`: `Towards front` (positive x direction) or `Towards right` (positive y direction).
+- `Use for control`: [Tilt servo used for yaw/pitch](#tilt-servos-for-yaw-pitch-control)
+  - `None`: Torque control is not used.
+  - `Yaw`: Tilt servos used to control yaw.
+  - `Pitch`: Tilt servos used to control pitch.
+  - `Both Yaw and Pitch`: Tilt servos are used to control both yaw and pitch.
+
+
+#### Tilt Servo Coordinate System
+
+The coordinate system for tilt rotor angles is shown below.
+The reference direction for tilt angles is straight upwards (0 degrees).
+Tilt angles towards the front or right of the vehicle are positive, and towards the back or to the left are negative.
+
+![Tilt Axis](../../assets/config/actuators/tilt_axis.png)
+
+The `Angle at min tilt` and `Angle at max tilt` indicate the range of movement for the tilt servo.
+The minimum tilt is the smaller _numerical value_ (not absolute) of the two angles.
+
+If the max/min tilt vectors are **P<sub>0</sub>** and **P<sub>1</sub>** as shown above, both tilt angles are positive but **θ<sub>0</sub>** is smaller:
+- `Angle at min tilt` = **θ<sub>0</sub>**
+- `Angle at max tilt` = **θ<sub>1</sub>**
+
+:::note
+If the diagram was mirrored so that **P<sub>0</sub>** and **P<sub>1</sub>** were tilting into the -x, -y quadrant, then both the tilt angles would be negative.
+Because **θ<sub>1</sub>** would more negative (smaller) than **θ<sub>0</sub>**, it would be the `Angle at min tilt`.
+
+Similarly, a servo that moves:
+- between the upright and forward postions would have `min=0` and `max=90`. 
+- symmetrically 45 degrees around the upright position would have `min=-45` and `max=45`
+- between the upright and backward positions would have `min=-90` and `max=0`.
+:::
+
+The `Tilt direction` indicates whether the servo tilts in the plane towards the `Front` or `Right` of the vehicle.
+On the diagram this would be represented by **α** that can only take values of 0 (front) or 90 (right).
```

Comment:
The values might be wrong. My assumption is that tilt direction is 0 pointing forward, increasing positive in rotation towards the right.

Reply:
Sounds correct to me.